galaxy_ng/app/api/ui/v2/permissions.py

```diff
@@ -16,3 +20,77 @@ def has_permission(self, request, view):
             return True
         # Otherwise, deny access
         return False
+
+
+class ComplexUserPermissions(AnsibleBaseUserPermissions):
```

Comment:
My current thought - it might be safer to parent from the existing `AccessPolicyBase`, as opposed to `AnsibleBaseUserPermissions`. The `has_object_permission` doesn't defer to the super class at all right now. That concerns me somewhat. I believe you have decent rules for editing the `is_superuser` field, but even with that established, it looks like edits to unrelated field (like, say, email, or first_name) are restricted to superuser. I don't know what the prior policies were, but I'm worried about throwing those out and using the DAB policies right now, when all the other access checks are done using the pulp access control models. Also, as written, I don't think `has_object_permission` will have any consistency with `has_permission`